packages/ra-no-code/src/builders/CreateBuilder.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+    CreateContextProvider,
+    CreateControllerProps,
+    useCreateContext,
+    useCreateController,
+} from 'ra-core';
+import { CreateProps, CreateView, SimpleForm } from 'ra-ui-materialui';
+import { getInputFromFieldDefinition } from './getInputFromFieldDefinition';
+import { useResourceConfiguration } from '../ResourceConfiguration';
+
+export const CreateBuilder = (props: CreateProps) => {
+    const controllerProps = useCreateController(props);
+
+    return (
+        <CreateContextProvider value={controllerProps}>
```

Comment:
why don't you use `<CreateBase>`?

Reply:
Forgot about it

Reply:
Even simpler, I'll use the `Create`, `Edit` and `List` directly